litellm/__init__.py

```diff
@@ -852,7 +855,28 @@ def add_known_models():
     "petals-team/StableBeluga2",
 ]
 
-ollama_models = ["llama2"]
+# Since Ollama models are local, it is inexpensive to refresh the list at startup.
+# Also, this list can change very quickly.
+
+ollama_dir = os.environ.get("OLLAMA_MODELS", None)
```

Comment:
This is the canonical way that the directory to use might be specified, though when not set has the home dir default.